Xamarin.Forms.Platform.iOS/Platform.cs

```diff
@@ -21,7 +21,7 @@ public class Platform : BindableObject, INavigation, IDisposable
 #if DEBUG
 				if (oldvalue != null && newvalue != null)
 				{
-					Log.Warning("Renderer", $"{bindable} already has a renderer attached to it: {oldvalue}. Please figure out why and then fix it.", nameof(oldvalue));
+					Log.Warning("Renderer", $"{bindable} already has a renderer attached to it: {oldvalue}. Please figure out why and then fix it.");
```

Comment:
o_O that's not very useful, but I guess it's all right, since customers will never see it.